src/mctp-netlink.h

```diff
@@ -8,6 +8,12 @@
 
 #include "mctp.h"
 
+enum endpoint_role {
+	ENDPOINT_ROLE_UNKNOWN,
+	ENDPOINT_ROLE_BUS_OWNER,
+	ENDPOINT_ROLE_ENDPOINT,
+};
+
```

Comment:
This is specific to the mctpd usage of the netlink code. This should be private to mctpd.c.